backend/controllers/bulkAppointmentsFormRedirect.js

```diff
@@ -1,5 +1,3 @@
 module.exports = (req, res) => {
```

Comment:
This is a strange module. Where is it used?

Reply:
It was used previously to clear out the session data and do the redirect to the add appointments.  Looking at it now, I don't think it's needed.  We can just redirect to the form directly as the session has already been cleared out.  What do you reckon?

Reply:
Yeah, my preference is to remove entirely.